tests/app/service/test_rest.py

```diff
@@ -2491,95 +2491,159 @@ def test_get_email_reply_to_addresses_with_multiple_email_addresses(client, noti
     assert not json_response[1]['updated_at']
 
 
-def test_add_service_reply_to_email_address(client, sample_service):
-    data = json.dumps({"email_address": "new@reply.com", "is_default": True})
-    response = client.post('/service/{}/email-reply-to'.format(sample_service.id),
-                           data=data,
-                           headers=[('Content-Type', 'application/json'), create_authorization_header()])
+@freeze_time("2016-01-01 11:09:00.061258")
```

Comment:
Do we need `freeze_time` for this test?

Reply:
Ahh, you are right, we don't, I will update it now 👍